src/features/broker-protection/actions/captcha.js

```diff
@@ -80,14 +81,16 @@ export function getCaptchaInfo (action) {
 * Takes the solved captcha token and injects it into the page to solve the captcha
 *
 * @param action
+* @param {string} token
+* @param {Document} root
```

Comment:
Does this also need to be `{Document|HTMLElement}`?

Reply:
Actually `HTMLElement` in this 1 particular case doesn't work - because `getElementsByName` is only on Document.

We should change the body of this function later to improve this, but for now I don't want to alter anything other than the params, so we can follow up :)

Reply:
Sounds perfect, thanks for the explanation.